applications/luci-app-acme/htdocs/luci-static/resources/view/acme.js

```diff
@@ -21,120 +31,455 @@ return view.extend({
 		o = s.tab("advanced", _('Advanced Settings'));
 
 		o = s.taboption('general', form.Flag, "enabled", _("Enabled"));
 		o.rmempty = false;
 
-		o = s.taboption('general', form.Flag, "use_staging", _("Use staging server"),
-			_("Get certificate from the Letsencrypt staging server " +
-				"(use for testing; the certificate won't be valid)."));
-		o.rmempty = false;
-		o.modalonly = true;
-
-		o = s.taboption('general', form.ListValue, "keylength", _("Key size"),
-			_("Key size (and type) for the generated certificate."));
-		o.value("2048", _("RSA 2048 bits"));
-		o.value("3072", _("RSA 3072 bits"));
-		o.value("4096", _("RSA 4096 bits"));
-		o.value("ec-256", _("ECC 256 bits"));
-		o.value("ec-384", _("ECC 384 bits"));
-		o.default = "2048";
-		o.rmempty = false;
-
 		o = s.taboption('general', form.DynamicList, "domains", _("Domain names"),
 			_("Domain names to include in the certificate. " +
 				"The first name will be the subject name, subsequent names will be alt names. " +
 				"Note that all domain names must point at the router in the global DNS."));
 		o.datatype = "list(string)";
+		o.rmempty = false;
 
-		if (stats[1].type === 'file') {
-			o = s.taboption('general', form.Flag, "update_uhttpd", _("Use for uhttpd"),
-				_("Update the uhttpd config with this certificate once issued " +
-					"(only select this for one certificate). " +
-					"Is also available luci-app-uhttpd to configure uhttpd form the LuCI interface."));
-			o.rmempty = false;
-			o.modalonly = true;
-		}
-
-		if (stats[0].type === 'file') {
-			o = s.taboption('general', form.Flag, "update_nginx", _("Use for nginx"),
-				_("Update the nginx config with this certificate once issued " +
-					"(only select this for one certificate). " +
-					"Nginx must support ssl, if not it won't start as it needs to be " +
-					"compiled with ssl support to use cert options"));
-			o.rmempty = false;
-			o.modalonly = true;
-		}
-
-		o = s.taboption('challenge', form.ListValue, "validation_method", _("Validation method"),
+		o = s.taboption("general", form.ListValue, "validation_method", _("Validation method"),
 			_("Standalone mode will use the built-in webserver of acme.sh to issue a certificate. " +
 			"Webroot mode will use an existing webserver to issue a certificate. " +
 			"DNS mode will allow you to use the DNS API of your DNS provider to issue a certificate."));
 		o.value("standalone", _("Standalone"));
 		o.value("webroot", _("Webroot"));
 		o.value("dns", _("DNS"));
-		o.default = "standalone";
+		o.default = "webroot";
 
-		o = s.taboption('challenge', form.Value, "webroot", _("Webroot directory"),
+		o = s.taboption("challenge_webroot", form.Value, "webroot", _("Webroot directory"),
 			_("Webserver root directory. Set this to the webserver " +
 				"document root to run Acme in webroot mode. The web " +
-				"server must be accessible from the internet on port 80."));
+				"server must be accessible from the internet on port 80.") + "<br/>" +
```

Comment:
Must be

`+ "<br />" +`

Reply:
the `<br/>` without a space should be fine. Minor style issue, let's leave it as is.